openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfile.java

```diff
@@ -660,6 +665,55 @@ public MatrixResult computeMatrix(MatrixRequest req) throws Exception {
         return mtxResult;
     }
 
+    public CentralityResult computeCentrality(CentralityRequest req) throws Exception {
+        CentralityResult res = new CentralityResult();
+
+        GraphHopper gh = getGraphhopper();
+        String encoderName = RoutingProfileType.getEncoderName(req.getProfileType());
+        FlagEncoder flagEncoder = gh.getEncodingManager().getEncoder(encoderName);
+        Graph graph = gh.getGraphHopperStorage().getBaseGraph();
+
+        HintsMap hintsMap = new HintsMap();
+        int weightingMethod = WeightingMethod.FASTEST;
+        setWeighting(hintsMap, weightingMethod, req.getProfileType(), false);
+        Weighting weighting = new ORSWeightingFactory().createWeighting(hintsMap, flagEncoder, gh.getGraphHopperStorage());
+
+        // filter graph for nodes in Bounding Box
+        LocationIndex index = gh.getLocationIndex();
+        NodeAccess nodeAccess = graph.getNodeAccess();
+        BBox bbox = req.getBoundingBox();
+        List<Integer> excludeNodes = req.getExcludeNodes();
+
+        ArrayList<Integer> nodesInBBox = new ArrayList<>();
+        index.query(bbox, new LocationIndex.Visitor() {
+            @Override
+            public void onNode(int nodeId) {
+                if (!excludeNodes.contains(nodeId)) {
+                    if (bbox.contains(nodeAccess.getLat(nodeId), nodeAccess.getLon(nodeId))) {
+                        nodesInBBox.add(nodeId);
+                    }
+                }
+            }
+        });
+
+        CentralityAlgorithm alg = new BrandesCentralityAlgorithm();
+        alg.init(graph, weighting);
+
+        HashMap<Integer, Double> betweenness = alg.compute(nodesInBBox);
+
+        //System.out.printf("Working on %d nodes, result has %d nodes\n, nodesInBBox.size(), betweenness.size());
```

Comment:
Please remove commented out code. If you need such an output for debugging, consider using an appropriate logger.